src/app/admin/components/learning-object-list-item/learning-object-list-item.component.html

```diff
@@ -35,22 +35,18 @@
 <clark-context-menu *ngIf="meatballOpen" [anchor]="meatballMenuElement" [offset]="{ top: 2, left: 10 }" (close)="toggleContextMenu()">
   <div #contextMenu>
     <ul (activate)="toggleContextMenu($event)">
-      <a *ngIf="learningObject.status !== 'released'" [routerLink]="['/onion/learning-object-builder', learningObject.id]" target="_blank"><li><i class="far fa-pencil"></i>Edit</li></a>
-      <a *ngIf="learningObject.revision && learningObject.revision?.status !== 'unreleased'" [routerLink]="['/onion/learning-object-builder', learningObject.id]" target="_blank"><li><i class="far fa-pencil"></i>Edit Revision</li></a>
+      <li *ngIf="learningObject.status !== 'released'" (click)="goToUrl('builder')" target="_blank"><i class="far fa-pencil"></i>Edit</li>
+      <li *ngIf="learningObject.revision && learningObject.revision?.status !== 'unreleased'" (click)="goToUrl('builder')"><i class="far fa-pencil"></i>Edit Revision</li>
+      <li (click)="toggleHierarchyBuilder(true)" *ngIf="learningObject.length !== 'nanomodule' && !hasChildren"><i class="fa-lg fal fa-chart-network"></i>Generate Hierarchy</li>
       <li *ngIf="learningObject.revision?.status === 'unreleased'" (click)="toggleRevisionDelete(true)"><i class="far fa-trash-alt"></i>Delete Revision</li>
-      <a [routerLink]="['/users', learningObject.author.username]" target="_blank"><li><i class="far fa-user"></i>View Author Contact</li></a>
-      <a [routerLink]="['/details', learningObject.author.username, learningObject.cuid]" target="_blank"><li><i class="far fa-cube"></i>View details</li></a>
+      <li (click)="goToUrl('contact')"><i class="far fa-user"></i>View Author Contact</li>
+      <li (click)="goToUrl('details')"><i class="far fa-cube"></i>View details</li>
       <li (click)="toggleRelevancyDate(true)"><i class="fal fa-history"></i>Mark Next Check Date</li>
       <li (click)="toggleAddEvaluatorModal(true)"><i class="fal fa-user-plus"></i>Assign Evaluators</li>
       <li *ngIf="hasParents === false" (click)="toggleChangeAuthorModal(true)"><i class="far fa-user"></i>Change Author</li>
       <li *ngIf="learningObject.status !== 'released' && (auth.user.accessGroups.includes('admin') || auth.user.accessGroups.includes('editor'))" (click)="toggleChangeCollectionModal(true)"><i class="fal fa-exchange"></i>Change Collection</li>
       <li *ngIf="learningObject.status === 'released' && (auth.user.accessGroups.includes('admin') || auth.user.accessGroups.includes('editor'))" (click)="toggleUnreleaseConfirm(true)"><i class="far fa-eye-slash"></i>Unrelease</li>
-      <a [routerLink]="['/onion/relevancy-builder', learningObject.id]" target="_blank">
-        <li class="new-feature_wrapper">
-          <p class="list-item_title"><i class="far fa-puzzle-piece"></i>Map & Tag</p>
-          <p class="new-item_element"><i class="fas fa-asterisk"></i>NEW</p>
```

Comment:
This can be commented out and saved somewhere for future use

Reply:
I just removed the wrapper. The styling is still all there